src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.ts

```diff
@@ -75,6 +75,21 @@ export class DotEditLayoutService {
         return this.getDotLayoutBody(this.getDotLayoutGridBox(grid));
     }
 
+    /**
+     * Take an array of DotPageContainer and return a DotContainerColumnBox.
+     *
+     * @param {DotPageContainer[]} containers
+     * @returns {DotContainerColumnBox[]}
+     */
+    getDotLayoutSidebar(containers: DotPageContainer[]): DotContainerColumnBox[] {
+        return containers.map((dotPageContainer: DotPageContainer) => {
+            return {
+                container: this.templateContainersCacheService.get(dotPageContainer.identifier),
+                uuid: dotPageContainer.uuid ? dotPageContainer.uuid : new Date().getTime().toString()
```

Comment:
remove this: new Date().getTime().toString()
with the change with the UUID in BE it is not neccessary

Reply:
done